app/modules.go

```diff
@@ -235,6 +237,11 @@ func simulationModules(
 ) []module.AppModuleSimulation {
 	appCodec := encodingConfig.Marshaler
 
+	// appModules := appModules(app, encodingConfig, skipGenesisInvariants)
```

Comment:
can we just remove this if we're commenting it out?

Reply:
ah yeah, that was purely there for getting CI cache to run, I moved some of my WIP code from another branch here lol

Reply:
deleted!